apiserver/facades/client/modelmanager/modelinfo_test.go

```diff
@@ -1338,3 +1340,13 @@ func (m *mockMigration) StartTime() time.Time {
 func (m *mockMigration) EndTime() time.Time {
 	return m.end
 }
+
+type mockModelManagerService struct{}
```

Comment:
We might as well generate a mock for this. This is really a stub, as we're not verifying the calls anywhere. Ultimately we'll have to.

Reply:
Although I am in agreement that this should be a mock, I think for this patch we should just replicate what we have. The whole model manager needs a re-work from a testing perspective and it seems like a big task to just do that in this patch.